CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ Types of changes:
 
 ## [Unreleased]
 
+## Fixed
+- Breadcrumbs activity check on `FragmentActivity` instead of `AppComponentActivity`
```

Comment:
This pull request changes a whole lot more than just fixing the Breadcrumbs (e.g. updated dependencies, build config changes etc). The rest of those changes should be listed here as well. 

In any case, I think it's a good idea to keep pull requests focused on one specific issue at the time instead of including a bunch of other off-topic changes as well.

Reply:
Have split up the PR into 2 separate ones.